pkgs/servers/code-server/default.nix

```
@@ -166,11 +176,24 @@ in stdenv.mkDerivation rec {
       -execdir ln -s ${ripgrep}/bin/rg {}/bin/rg \;
 
     # patch shebangs of everything, also cached files, as otherwise postinstall
-    # will not be able to find /usr/bin/env, as it does not exists in sandbox
+    # will not be able to find /usr/bin/env, as it does not exist in sandbox
     patchShebangs .
 
-    # rebuild binaries, we use npm here, as yarn does not provider alternative
-    # that would not atempt to try to reinstall everything and break out
+    '' + stdenv.lib.strings.optionalString stdenv.isDarwin ''
+
```

```suggestion
    '' + lib.optionalString stdenv.isDarwin ''
```

not sure if optionalString or optionalStrings